src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -63,7 +61,7 @@ public function showAction(Request $request, FlattenException $exception, DebugL
         $code = $exception->getStatusCode();
 
         return new Response($this->twig->render(
-            (string) $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
+            $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
```

Comment:
@fabpot FOSRestBundle is using it (it was the reason it was introduced at first btw)